adminapp/src/pages/OfferingDetailPage.js

```diff
@@ -41,7 +42,47 @@ export default function OfferingDetailPage() {
               { label: "Created At", value: dayjs(offering.createdAt) },
               { label: "Opening Date", value: dayjs(offering.opensAt) },
               { label: "Closing Date", value: dayjs(offering.closesAt) },
+              {
+                label: "Begin Fulfillment At",
+                value: dayjs(offering.beginFulfillmentAt),
+              },
+              {
+                label: "Prohibit Charge At Checkout",
+                value: offering.prohibitChargeAtCheckout ? "Yes" : "No",
+              },
+              {
+                label: "Image",
+                value: (
+                  <SumaImage
+                    image={offering.image}
+                    alt={offering.image.name}
+                    className="w-100"
+                    params={{ crop: "center" }}
+                    h={225}
+                    width={225}
+                  />
+                ),
+              },
               { label: "Description", value: offering.description },
+              { label: "Fulfillment Prompt", value: offering.fulfillmentPrompt },
+              {
+                label: "Fulfillment Confirmation",
+                value: offering.fulfillmentConfirmation,
+              },
+            ]}
+          />
+          <RelatedList
+            title="Fulfillment Options"
+            rows={offering.fulfillmentOptions}
+            headers={["Id", "Description", "Type", "Address"]}
+            keyRowAttr="id"
+            rowClass={(row) => (row.closedAt ? classes.closed : "")}
+            toCells={(row) => [
+              row.id,
+              row.description,
+              row.type,
+              row.address &&
+                `${row.address.address1} ${row.address.address2} ${row.address.city}, ${row.address.stateOrProvince} ${row.address.postalCode}`,
```

Comment:
Create a `oneLineAddress.js` module/function, based on `Address#one_line_address` in Ruby (the main problem here is that if there's no `address2` you end up with `123 Mai St  Portland, OR 97215` (two spaces between address1 and city).